gpii/node_modules/transformer/test/TransformerTests.js

```diff
@@ -1551,3 +1551,62 @@ jqUnit.test("gpii.transformer.hasSupportedSettings() tests", function () {
         jqUnit[oneTest.expected ? "assertTrue" : "assertFalse"]("gpii.transformer.hasSupportedSettings() test - " + oneTest.name, result);
     });
 });
+
+var jsonArrayToTaggedArrayPayload = {
+    value: [
+        "Buttons",
+        "Lists",
+        "Menus",
+        "Text",
+        "Other"
+    ],
+    spec: {
+        tag: "ReadItem",
+        elems: [
+            "Buttons",
+            "Lists",
+            "Menus",
+            "Text",
+            "Other"
+        ],
+        hasElemVal: {
+            "isTrue": "true",
+            "isFalse": "false"
+        }
+    }
+};
+
+jqUnit.test("gpii.transformer.jsonArrayToTaggedArray() tests", function () {
+    var result = gpii.transformer.jsonArrayToTaggedArray(jsonArrayToTaggedArrayPayload.value,
+        jsonArrayToTaggedArrayPayload.spec);
+
+    var expRes = [
+        {
+            "$t": "Buttons",
+            "ReadItem": "true"
```

Comment:
Supply test cases which actually execute the "false" branch!

Reply:
Done too!